documentation/asciidoc/computers/configuration/device-tree.adoc

```diff
@@ -991,3 +991,121 @@ Further ports can be controlled by the DT, for more details see <<part3,section
 ==== Further help
 
 If you've read through this document and not found the answer to a Device Tree problem, there is help available. The author can usually be found on Raspberry Pi forums, particularly the https://forums.raspberrypi.com/viewforum.php?f=107[Device Tree] forum.
+
+[[part5.0]]
```

Comment:
minor nitpick, but for consistency this should be `part5` rather than `part5.0`

Reply:
done